backends/ebpf/CMakeLists.txt

```diff
@@ -56,8 +56,8 @@ execute_process(COMMAND uname -r
   OUTPUT_STRIP_TRAILING_WHITESPACE
   RESULT_VARIABLE rc)
 message(STATUS "Detected kernel version: ${P4C_EBPF_KERNEL_VER}")
-if (EXISTS /usr/src/linux-headers-${P4C_EBPF_KERNEL_VER}/include/uapi/linux)
-  include_directories(/usr/src/linux-headers-${P4C_EBPF_KERNEL_VER}/include/uapi/linux)
+if (EXISTS /usr/src/linux-headers-${P4C_EBPF_KERNEL_VER}/include)
+  include_directories(/usr/src/linux-headers-${P4C_EBPF_KERNEL_VER}/include)
```

Comment:
I'm afraid this is not going to include the right header that EBPF needs.
Can out try:
```if (EXISTS /usr/src/linux-headers-${P4C_EBPF_KERNEL_VER}/include)
  include_directories(/usr/src/linux-headers-${P4C_EBPF_KERNEL_VER}/include /usr/src/linux-headers-${P4C_EBPF_KERNEL_VER}/include/uapi/linux)
```

Reply:
What is the error? The compilation should never fail, running the tests may fail.

Reply:
See this comment on this same PR: https://github.com/p4lang/p4c/pull/2049#issuecomment-540106445
for attached files with what I think is a slightly different change suggested by Calin, and the error message I see when I attempt to build p4c with that different change.